go/bind/keybase.go

```diff
@@ -127,6 +127,12 @@ func Init(homeDir string, logFile string, runModeStr string, accessGroupOverride
 		Logs:         logs,
 	}
 
+	// open the connection
```

Comment:
Perhaps nothing, but I thought it would be a good idea to make sure the connection is open before the goroutine starts for KBFS init.  Looked like possible race condition.